analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd

```diff
@@ -244,17 +246,235 @@ compiled_df <- compiled_df %>%
 
 The `molecular-subtyping-EPN` module has not been completed yet, but the logic that is in that module may mean that we need to include revising the labels of `PT_AQWDQW27`.
 
+```{r}
+compiled_df %>%
+  filter(Kids_First_Participant_ID == "PT_AQWDQW27")
+# This sample is missing from the EPN table, but it should be there - will have to investigate and update this later.
+```
+
+### `PT_6Q0NPVP3`
+
+The specimens for this patient, BS_5JM573JC and BS_E5H6CFYT, were classified as HGAT due to the presence of a histone mutation, but with the removal of LGAT from the HGAT module, this sample will no longer show up in two modules.
+```{r}
+compiled_df %>%
+  filter(Kids_First_Participant_ID == "PT_6Q0NPVP3")
+```
+
+### Are there any other duplicate subtypes?
+```{r}
+unique_subtypes <- compiled_df %>%
+  select(Kids_First_Participant_ID, sample_id, molecular_subtype) %>%
+  distinct()
+
+unique_subtypes[duplicated(unique_subtypes$sample_id),]
+#PT_KTRJ8TFY (fixed in clinical feedback) and PT_6Q0NPVP3 (fixed in HGG module removing LGAT)
+```
+### `PT_00G007DM`
+
```

Comment:
```suggestion
7316-272 is subtyped as ETMR because a C19MC alteration was found and pathology rerview confirms the new diagnosis

```{r}
compiled_df <- compiled_df %>%
  mutate(molecular_subtype= case_when(sample_id == "7316-272"~"ETMR, C19MC-altered",
                                      TRUE ~ molecular_subtype),
         short_histology= case_when(sample_id == "7316-272"~"ETMR",
                                      TRUE ~ short_histology),
         integrated_diagnosis = case_when(sample_id == "7316-272"~"Embryonal tumor with multilayer rosettes, C19MC-altered",
                                      TRUE ~ integrated_diagnosis)
  ) %>%
  unique()

```

Sample 7316-272 from PT_00G007DM has both MB (to be classified) and ETMR subtyping values in compiled_df. Since we know that C19MC alteration is found in this sample we would need to update the molecular_subtype/short_histology/intergated_diagnosis values to ETMR to not have duplicated rows for this sample :

Kids_First_Participant_ID<chr> | sample_id<chr> | Kids_First_Biospecimen_ID<chr> | molecular_subtype<chr> | integrated_diagnosis<chr> |  
-- | -- | -- | -- | -- | --
PT_00G007DM | 7316-272 | BS_QWNBZ9RJ | ETMR, C19MC-altered | Embryonal tumor with multilayer rosettes, C19MC-altered |  
PT_00G007DM | 7316-272 | BS_K07KNTFY | ETMR, C19MC-altered | Embryonal tumor with multilayer rosettes, C19MC-altered |  
PT_00G007DM | 7316-272 | BS_QWNBZ9RJ | MB, To be classified | NA |  
PT_00G007DM | 7316-272 | BS_K07KNTFY | MB, To be classified | NA